components/PoeModal.tsx

```diff
@@ -0,0 +1,659 @@
+                    <div className="mb-2 mt-4 self-start text-2xl font-medium text-white">
+                      The wallet {"can't"} confirm that the transaction has been
+                      validated. You can check it in{" "}
+                      <a
+                        className="text-zinc-200 hover:text-zinc-300"
+                        href={`https://${
+                          PREFERED_NETWORK === NetworkType.GHOSTNET
+                            ? "ghostnet."
+                            : ""
+                        }tzkt.io/${timeoutAndHash[1]}`}
+                        target="_blank"
+                        rel="noreferrer"
+                      >
+                        the explorer
+                      </a>
+                      , and if it is, {"it'll"} appear in the proposals
+                    </div>
+                    <button
+                      className="mt-6 rounded border-2 bg-transparent p-2 font-medium text-white hover:outline-none"
+                      onClick={reset}
+                    >
+                      Close
+                    </button>
+                  </div>
+                );
+
+              if (!!transactionError)
+                return (
+                  <div className="col-span-2 flex w-full flex-col items-center justify-center">
+                    <div className="mb-2 mt-4 text-center text-xl font-medium text-white">
+                      {transactionError}
+                    </div>
+                    <button
+                      className="mt-6 rounded border-2 bg-transparent p-2 font-medium text-white hover:outline-none"
+                      onClick={reset}
+                    >
+                      Close
+                    </button>
+                  </div>
+                );
+
+              if (!timeoutAndHash[0] && !!timeoutAndHash[1])
+                return (
+                  <div className="col-span-2 flex w-full flex-col items-center justify-center">
+                    <p>Succesfully created the proposal!</p>
+                    <button
+                      className="mt-6 rounded border-2 bg-transparent p-2 font-medium text-white hover:outline-none"
+                      onClick={reset}
+                    >
+                      Close
+                    </button>
+                  </div>
+                );
+
+              if (!transfers) return null;
+
+              return (
+                <>
+                  <div className="col-span-2 flex w-full flex-col items-center justify-center">
+                    <div className="mb-2 mt-4 self-start text-2xl font-medium text-white">
+                      Incoming action{(transfers?.length ?? 0) > 1 ? "s" : ""}{" "}
+                      from {currentMetadata?.[1].name}
+                    </div>
+                    <p className="self-start text-sm text-zinc-400">
+                      <Alias address={address ?? ""} /> will create the proposal
```

Comment:
I'm thinking that put a warning message when `address` isn't equally to current address to reduce confusing ?